src/einsteinpy/symbolic/ricci.py

```diff
@@ -96,3 +152,44 @@ def symbols(self):
 
         """
         return self.syms
+
+
+class RicciScalar:
```

Comment:
In truth, this actually has no need to be a class since this does not represent a data type.

Reply:
I know, but initially it was raised like this. If they prefer that I change it, I will do it